apps/examples/testcase/ta_tc/task_manager/itc/itc_task_manager_main.c

```diff
@@ -515,7 +515,7 @@ void itc_task_manager_unicast_thrice_p(void)
 }
 
 
-void itc_broadcast_handler(void *user_data, void *info)
+void itc_broadcast_handler(tm_msg_t *user_data, tm_msg_t *info)
```

Comment:
Line 535, you set the broadcast cb function as this. Is there any special reason for this?

Reply:
When the broadcast happens the second time it should go to the else part in this function.

Reply:
As you changed the type of input parameter, usage of the input parameter should be changed.
Such as `(char *)info` will be `(char *)info->msg`.

Reply:
Changed that